source/MaterialXTest/GenShader.cpp

```diff
@@ -39,6 +39,23 @@ namespace mx = MaterialX;
 // Base tests
 //
 
+TEST_CASE("GenShader: Utilities", "[genshader]")
+{
+    // Test simple text substitution
+    std::string test1 = "Look behind you, a $threeheaded $monkey!";
```

Comment:
I like the phrases. I would have never have been able to come up with something like these :)

Reply:
I can't take credit, it's from the game Monkey Island (all time favorite adventure game) :)